content/200-concepts/100-components/02-prisma-client/058-transactions.mdx

```diff
@@ -104,46 +102,70 @@ generator client {
 
 Then you can pass an async function into [`$transaction`](../../../guides/performance-and-optimization/prisma-client-transactions-guide#transaction-api).
 
-The following is an example of purchasing a movie ticket online. In this example, we really don't want to sell the same ticket twice:
+Let's say we're building an online banking system. One of the actions we want to perform is to send money from one person to another person.
```

Comment:
```suggestion
Imagine that you are building an online banking system. One of the actions to perform is to send money from one person to another.
```
cf. https://www.prisma.io/docs/about/prisma-docs/style-guide#use-you-as-the-pronoun